packages/cli/src/__tests__/dotenv-5-only-root.test.ts

```diff
@@ -1,11 +1,12 @@
-import { jestConsoleContext, jestContext } from '@prisma/sdk'
+import { jestConsoleContext, jestContext, loadEnvFileAndPrint } from '@prisma/sdk'
 
 const ctx = jestContext.new().add(jestConsoleContext()).assemble()
 
 it('should not load root .env file', async () => {
-  process.argv.push('--version')
   ctx.fixture('dotenv-5-only-root')
-  await import('../bin')
-  expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+  loadEnvFileAndPrint()
```

Comment:
Does using bin not work anymore, even by passing --schema?

Reply:
The env loading logic was done directly in `bin.ts` before and now it's moved to each command because not all commands need to load them (and therefore print)

Reply:
Thanks, makes sense!